packages/office-ui-fabric-react/src/components/Dialog/Dialog.tsx

```diff
@@ -51,49 +56,49 @@ export class Dialog extends BaseComponent<IDialogProps, IDialogState> {
       onDismiss,
       onDismissed,
       onLayerDidMount,
-      onLayerMounted,
       responsiveMode,
       subText,
       title,
       type,
       contentClassName,
-      topButtonsProps
+      topButtonsProps,
+      dialogContentProps,
+      modalProps,
+      containerClassName,
+      hidden
     } = this.props;
     let { id } = this.state;
 
-    const dialogClassName = css(this.props.className, {
-      ['ms-Dialog--lgHeader ' + styles.isLargeHeader]: type === DialogType.largeHeader,
-      ['ms-Dialog--close ' + styles.isClose]: type === DialogType.close,
-    });
-    const containerClassName = css(this.props.containerClassName, styles.main);
-
     return (
       <Modal
-        className={ dialogClassName }
-        containerClassName={ containerClassName }
         elementToFocusOnDismiss={ elementToFocusOnDismiss }
         firstFocusableSelector={ firstFocusableSelector }
         forceFocusInsideTrap={ forceFocusInsideTrap }
         ignoreExternalFocusing={ ignoreExternalFocusing }
         isBlocking={ isBlocking }
         isClickableOutsideFocusTrap={ isClickableOutsideFocusTrap }
         isDarkOverlay={ isDarkOverlay }
-        isOpen={ isOpen }
-        onDismiss={ onDismiss }
+        isOpen={ isOpen ? isOpen : !hidden }
```

Comment:
oh, another issue is that isOpen is set to false by default (or was in the past and should probably should continue to be). So I really should be checking to see if user specified hidden or not, and use that prop.